test/boost/sstable_compaction_test.cc

```diff
@@ -2383,8 +2383,8 @@ SEASTAR_THREAD_TEST_CASE(scrub_validate_mode_validate_reader_test) {
         frags.emplace_back(make_partition_start(2));
         frags.emplace_back(make_partition_end());
 
-        const auto valid = scrub_validate_mode_validate_reader(make_flat_mutation_reader_from_fragments(schema, permit, std::move(frags)), *info).get();
-        BOOST_REQUIRE(valid);
+        const auto errors = scrub_validate_mode_validate_reader(make_flat_mutation_reader_from_fragments(schema, permit, std::move(frags)), *info).get();
+        BOOST_REQUIRE(errors == 0);
```

Comment:
`BOOST_REQUIRE_EQUAL(errors, 0);`

This way the value of `errors` will be printed if this fails.